### DISCUSSION TITLE
Help with Google Cloud - menu missing but nightscout working

### DISCUSSION THREAD

**Discussion by analogmonster** [NightscoutFoundation/xDrip#3058] [Q&A] [answered]
Hi, afraid I cannot post an image of the status page because I cannot access it.

My CGM is BYODA (G6 INT_v1.10.0.2_android9_MMOL), my uploader is AAPS (v3.0.0.1).

I migrated to google cloud 6 months ago.  Noticed yesterday my nightscout was unavailable.

Availability issues were caused by freeDNS (not signed in for 6 months).  While I was there I also changed my subdomain to one considered low risk.

I used SSH to get to the menu.  I cannot remember exactly what it was called, but I ran the option to change freeDNS subdomain and that went through fine.

I ran a menu option - Update Scripts.  Closed terminal and restarted.  Went to status.  Missing node_modules and packages.

I went to Google Cloud submenu and ran update platform.   Closed terminal and restarted.

Instead of the menu, I get a Ubuntu screen saying menu.sh: no such file or directory.  I am not that well versed in linux, but maybe someone can point me in the right direction and I can either get into the menu or at least make a backup.

Wish I could say I was smart enough to do a backup first!    Both my old and new subdomains now show my nightscout with latest CGM data, so I can tell it is running.  Something must have gone wrong when I was doing the update platform option.
![google cloud nightscout no menu](https://github.com/NightscoutFoundation/xDrip/assets/6210412/d4926841-f449-45da-b31e-3b921a8230a4)

**Comment by Navid200** [comment, marked as answer]
I'm sorry about the trouble.

Please copy the following line and paste it into the terminal and enter.
   
```curl https://raw.githubusercontent.com/jamorham/nightscout-vps/vps-1/bootstrap.sh | bash```   
   
It will bring up a message saying that you already have a system on this machine; are you sure you want to run bootstrap.
Confirm and let it run.  It will not harm your database.

After it completes, it will ask you to press enter to go to the status page.
Please let me see the status page then.
If I don't respond right away and you need to go, there is no problem.  You can just click on x at the top right corner of the SSH window and close it.
When you want to continue, after you click on SSH, it will bring up the menu and you are back you were after completing the bootstrap.

**Comment by analogmonster** [comment]
It is all working now and I have made a backup also, thanks for your help

**Reply by Navid200** [reply]
Great!
You're welcome.

The setup has gone through many changes.  
I didn't want to ask anyone to start from scratch to update.  The result is that depending on how long ago the system was put together, the update sequence may include seeing a black screen instead of the menu.  This can startle one even though it is expected.  

I need to document this better in the guide to avoid any inconvenient surprises.
